<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addToCartButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to the Bin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromCartButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove from Bin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceLineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter price of element</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter name of element</string>
    </property>
   </widget>
   <widget class="QComboBox" name="partComboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Engine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oil</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brake Disc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brake Pad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cooling System</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fuel System</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lights</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Radiator</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spark Plug</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suspension</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="nameToRemoveLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of element to delete</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cartTextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="totalPriceLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Price: 0$</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
